### PR TITLE
Purity RB

### PR DESCRIPTION
**Pull request by dieris**
Example sequence for purity RB

### CODE REVIEW

File: QGL/BasicSequences/RB.py [hunk 1]
```diff
@@ -150,29 +132,27 @@ def SingleQubitRB_AC(qubit, seqs, showPlot=False):
...
     """
     seqsBis = []
-    for seq in seqs:
-        seqsBis.append([DiAC(qubit, c, compiled) for c in seq])
-
-    #Add the measurement to all sequences
-    for seq in seqsBis:
-        seq.append(MEAS(qubit))
+    op = [Id(qubit, length=0), Y90m(qubit), X90(qubit)]
+    for ct in range(1+2*purity):
```
Comment (caryan) [inline QGL/BasicSequences/RB.py]: I think a ternary operator would clearer here: `3 if purity else 1`

File: QGL/BasicSequences/RB.py [hunk 2]
```diff
@@ -150,29 +132,27 @@ def SingleQubitRB_AC(qubit, seqs, showPlot=False):
...
+        for seq in seqs:
+            seqsBis.append([DiAC(qubit, c, compiled) for c in seq])
+            if purity:
```
Comment (caryan) [inline QGL/BasicSequences/RB.py]: since you've constrained the range above this if block is not necessary

File: QGL/BasicSequences/RB.py [hunk 3]
```diff
@@ -119,20 +107,14 @@ def TwoQubitRB(q1, q2, seqs, showPlot=False, suffix=""):
 
 
 def SingleQubitRB_AC(qubit, seqs, showPlot=False):
```
Comment (caryan) [inline QGL/BasicSequences/RB.py]: do we want purity here too or all we all in on DiAC?

Comment (dieris) [inline QGL/BasicSequences/RB.py]: I guess it makes sense to have the option everywhere, this was an experiment. I'll update

**Pull request merged**